packages/workspace/src/schematics/preset/preset.ts

```diff
@@ -41,7 +41,7 @@ function createPreset(options: Schema): Rule {
           name: options.name,
           style: options.style
         },
-        { interactive: false }
```

Comment:
By default, I believe this is inherited?

ie. If you run the `ng g @nrwl/workspace:preset` in `ng-new` with `--no-interactive` will these schematics inherit that?

Reply:
Yes, it will inherit that. Even with the `interactiveMode` set to true, if I run `ng g @nrwl/workspace:preset --interactiveMod=true --no-interactive`, there will be no prompts. I tried to use the `interactive` flag all the way through, but it never gets passed to the `preset` schematic as an option.